compute_sdk/globus_compute_sdk/sdk/_environments.py

```diff
@@ -27,6 +27,20 @@ def get_web_service_url(envname: str | None) -> str:
     return urls.get(env, urls["production"])
 
 
+def get_amqp_service_host(env_name: str | None = None) -> str:
+    env = env_name or _get_envname()
+    hosts = {
+        "production": "amqps.funcx.org",
+        "dev": "amqps.dev.funcx.org",
+        "preview": "compute.amqps.preview.globus.org",
```

Comment:
I hope you knew `preview`is different from the other non-prod envs from existing code for API url, and didn't waste time trying to debug the URL mismatch like I did!